SwiftIOSExperiments/Modules/Profile/Contract/ProfileInteractor.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ProfileInteractor.swift
+//  SwiftIOSExperiments
+//
+//  Created by James Santos on 15/2/17.
+//  Copyright © 2017 James Santos. All rights reserved.
+//
+
+import UIKit
+
+class ProfileInteractor: ProfilesUseCase {
+
+	weak var output: ProfilesInteractorOutput!
+
+	func fetchProfile(_ idToken: String) {
+		ProfileService.getUserInfo(idToken, completion: { [weak self] result in
+			DispatchQueue.main.async {
```

Comment:
i think you have to set your indentation to 2 space and limit with lint the number of character line with 130 char